docs/generate-time-series/javascript.rst

```diff
@@ -0,0 +1,232 @@
+.. _gen-ts-javascript:
+
+==========================================
+Generate time series data using Javascript
+==========================================
+
+This tutorial will show you how to generate some :ref:`experimental time series
+data <gen-ts>` by sampling your `system load`_ using Javascript, specifically `Node.js`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+CrateDB must be :ref:`installed and running <install-run>`.
+
+Make sure you're running up-to-date Node/npm.
```

Comment:
can you separate out Node and npm?

e.g., on one line, say:

```
Make sure you're running up-to-date `Node`_  (we recommend X or higher)..
```

link the name "Node" to the Node homepage. and I'd include a recommended version

then for npm, how about including a line that shows you how to upgrade npm to the latest version using npm itself? you could then also add a small fix to the python tutorial that does the same with pip

(please also link the first instance of "npm" to the npm homepage)

Reply:
I honestly have no idea what would be the recommended version here. I have the latest installed but I didn't try the tutorial out on all the other node versions.

Reply:
perhaps just this then:

```
Make sure you're running up-to-date `Node`_
```